src/test/regress/expected/isolation_append_copy_vs_all.out

```diff
@@ -0,0 +1,352 @@
+Parsed test spec with 2 sessions
```

Comment:
I added a description to README.md file and also added a header to direct readers to the README.md file